src/python/deepgnn/graph_engine/snark/converter/process.py

```diff
@@ -96,7 +83,7 @@ def converter_process(
         if line == FLAG_ALL_DONE:
             break
 
-        node = decoder.decode(line)
+        node = decoder.decode(line)  # type: ignore
```

Comment:
why do we need to start ignoring types here and it didn't happen before?

Reply:
Since decoder has to be optional to accept default values, this has to be added.